docs/setup_qa_pipeline.md

```diff
@@ -0,0 +1,57 @@
+# Steps to setup an end2end QA pipeline 
+Clone the Anserini, models, and data repo:
+```
+git clone https://github.com/rosequ/Anserini.git
+git clone https://github.com/castorini/data.git
+git clone https://github.com/castorini/models.git
```

Comment:
You don't need to have python code to make it a python package. We are just using pip and requirements.txt here as a module management tool.